ext/pdo/tests/pdo_dsn_containing_credentials.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+PDO Common: Pass credentials in dsn instead of constructor params
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo')) die('skip');
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+
+$driver = substr(getenv('PDOTEST_DSN'), 0, strpos(getenv('PDOTEST_DSN'), ':'));
+if (!in_array($driver, array('mssql','sybase','dblib','firebird','mysql','oci')))
+	die('not supported');
```

Comment:
@petk This needs to be 'skip: ' prefixed if you merge this PR

Reply:
cc @SjonHortensius :point_up: can you please only check this one out and we're good to go then I think. I'll check this out by the end of the day then...